### PR TITLE
Wrapped ccode and implemented class CodePrinter

### PR DESCRIPTION
**Pull request by ShikharJ**
Relevant: #150

### CODE REVIEW

**Comment by ShikharJ** [comment]
Ping @isuruf

**Comment by isuruf** [comment]
@ShikharJ, I was working on the same thing here, https://github.com/isuruf/symengine.py/tree/ccode2. Would you be interested in finishing that?

**Comment by ShikharJ** [comment]
@isuruf Sure.

**Comment by ShikharJ** [comment]
Updated @isuruf . Please take a look.

File: symengine/printing.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from symengine.lib.symengine_wrapper import ccode, _sympify
+
+class CCodePrinter:
+
+    def doprint(self, expr, assign_to=None):
+        if isinstance(assign_to, str):
+            if expr.is_Matrix:
+                #TODO: Implement MatrixSymbols
+                raise NotImplementedError
```
Comment (isuruf) [inline symengine/printing.py]: This case is handled in lines 24-28

**Comment by ShikharJ** [comment]
Updated @isuruf.

**Comment by isuruf** [comment]
We should write some tests.

**Comment by isuruf** [comment]
Thanks

**Pull request merged**